packaging/rest-service/files/opt/manager/scripts/create_system_filters.py

```diff
@@ -0,0 +1,72 @@
+#!/opt/manager/env/bin/python
+
+import os
+from datetime import datetime
+
+from manager_rest import constants
+from manager_rest.storage import models
+from manager_rest.storage.models_base import db
+from manager_rest.flask_utils import setup_flask_app
+
+
+def create_system_filters():
+    with setup_flask_app().app_context():
+        current_deployment_filters = db.session.query(models.DeploymentsFilter)
+        curr_dep_filters_ids = {dep_filter.id for dep_filter
+                                in current_deployment_filters}
+        if ({'csys-environment-filter', 'csys-service-filter'} ==
```

Comment:
This will lead to an interesting error if one of those exists but the other doesn't. Is that acceptable?

Reply:
Changed the script to be idempotent. I.e. each filter will be created only if it does not exist.